packages/core/test/invoke.test.ts

```diff
@@ -2985,6 +3005,7 @@ describe('invoke', () => {
         },
         {
           actors: {
+            // TODO: allow Behavior, not just BehaviorCreator
```

Comment:
If I read things correctly - this is outdated because I see Behaviors used directly.
```suggestion
```

Reply:
This (`() => fromCallback(...)`) is still a `BehaviorCreator`, so that still needs to be fixed.